Description: add support for abbreviated cidr addresses
 This issue was deemed "not a bug" by upstream, so the patch was not
 accepted
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=22373
Author: Bart Martens <bartm@debian.org>
Reviewed-by: Nick Morrott <knowledgejunkie@gmail.com>
Last-Update: 2021-04-11

--- a/lib/Net/CIDR.pm
+++ b/lib/Net/CIDR.pm
@@ -440,6 +440,7 @@
     ($isipv6, $ip)=_ipv6to4($iparg);
 
     my @ips= split (/\.+/, $ip);
+    for( my $i = $#ips + 1 ; $i < 4 ; $i++ ) { $ips[$i] = 0; }
 
     grep {
 	croak "$_, in $iparg, is not a byte" unless $_ >= 0 && $_ <= 255 && $_ =~ /^[0-9]+$/;
